spec/formatting.md

```diff
@@ -663,17 +663,44 @@ These are divided into the following categories:
     > Example invalid messages resulting in a Missing Fallback Variant error:
     >
     > ```
-    > match {$one}
+    > match {$one :func}
     > when 1 {Value is one}
     > when 2 {Value is two}
     > ```
     >
     > ```
-    > match {$one} {$two}
+    > match {$one :func} {$two :func}
     > when 1 * {First is one}
     > when * 1 {Second is one}
     > ```
 
+A **_Missing Selection Annotation error_** is an error that occurs when the _message_
+    contains a _selector_ that does not have an _annotation_,
+    or contains a _variable_ that does not directly or indirectly reference a _declaration_ with an _annotation_.
+
+    > Example invalid messages resulting in a Missing Selector Annotation error:
```

Comment:
This says "Missing Selector Annotation error" but the definition says "Missing Selection Annotation error". Choose one.

Also, the name should be italicized here.

Reply:
```suggestion
    > Example invalid messages resulting in a _Missing Selector Annotation error_:
```